ledger/participant-integration-api/src/main/scala/platform/store/backend/StorageBackend.scala

```diff
@@ -340,6 +341,15 @@ object DBLockStorageBackend {
   }
 }
 
+trait DebugStorageBackend {
```

Comment:
not sure about naming here...why debug? How about IntegrityStorageBackend?

Reply:
My intention was to create a trait that would collect all methods that are not to be used in production. I don't have strong opinion here, renamed to `IntegrityStorageBackend`.